codegen.yml

```diff
@@ -0,0 +1,36 @@
+schema: api-server/src/graphql/schema.graphql
+documents: web-app/src/**/*.graphql
+overwrite: true
+generates:
+  api-server/generated/graphql-resolvers.ts:
+    config:
+      contextType: api-server/src/graphql/resolvers/context#MyContext
+      mapperTypeSuffix: Model
+      mappers:
+        Resource: api-server/src/models/types#Resource
```

Comment:
Can you give a brief explanation or link me to a resource on what this code generation does? And also, if we add a new type on schema, does it also need to be added here manually? Maybe it's something that would happen automatically?